CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 * Add option `skipReplace` for `apos.doc.changeDocIds` method to skip the replacing of the "old" document in the database.
 * The `@apostrophecms/i18n` module now exposes a `locales` HTTP GET API to aid in implementation of native apps for localized sites.
 * Context menus can be supplied a `menuId` so that interested components can listen to their opening/closing.
+* Allow to set mode in `AposWidget` component through props.
```

Comment:
OK, but is this a situation where we'd be better off not using `AposWidget` at all to render the image? Don't we have the data to just render the image ourselves? I realize there may be a focal point and such